howtos/how-to-add-a-create_new_chat-button-in-talkjs/create-new-chat.js

```diff
@@ -0,0 +1,129 @@
+const $targetEl = document.getElementById('crud-modal');
+const modal = new Modal($targetEl);
+const appId = "<Add your App ID here!>";
+
+Talk.ready.then(function () {
+    const me = new Talk.User({
+        id: '123457',
+        name: 'Alice',
+        email: 'alice@example.com',
+        photoUrl: 'https://talkjs.com/images/avatar-1.jpg',
+        role: 'default',
+        welcomeMessage: 'Hey there! How are you? :-)',
+    });
+    window.talkSession = new Talk.Session({
+        appId: appId,
+        me: me,
+    });
+    const other = new Talk.User({
+        id: '654321',
+        name: 'Sebastian',
+        email: 'Sebastian@example.com',
+        photoUrl: 'https://talkjs.com/images/avatar-5.jpg',
+        role: 'default',
+        welcomeMessage: 'Hey, how can I help?',
+    });
+
+    const conversation = talkSession.getOrCreateConversation(
+        Talk.oneOnOneId(me, other)
+    );
```

Comment:
As we're adding more users to the conversation now it would make sense to not use `oneOnOneId` and instead go for something like

```suggestion
    const conversation = talkSession.getOrCreateConversation("createNewChatExampleConv");
```